lib/app/models/request_response.rb

```diff
@@ -14,6 +14,7 @@ class RequestResponse
       property :response_body, Text
       property :direction, String
       property :instance_id, String
+      property :sequence, Serial, unique_index: true
```

Comment:
No biggie, but I think something like `request_order` would be a better name.  "Sequence" already has a different meaning in inferno and when I first saw this while inspecting the database my mind was expecting something different.

Reply:
Yeah, `request_order` or `request_index` might make more sense for this, given that Sequence is already taken.

Reply:
I third this suggestion, and updated the column.  Could some verify my update real quick?

Reply:
Tested out the change and it worked as expected 👍